src/formBuilder/Card.test.js

```diff
@@ -222,5 +226,28 @@ describe('Card', () => {
 
     const inputTypeLabel = getHeadingText(wrapper, 3);
     expect(inputTypeLabel).toContain('Custom Input Type');
+
+    const placeHolderLabel = getHeadingText(wrapper, 3);
+    expect(placeHolderLabel).toContain('Custom Input Type');
+  });
+
+  it('calls the onChange on placeholder', () => {
```

Comment:
```suggestion
  it('allows a user to enter a custom placeholder', () => {
```

would describe the test better